src/SFA.DAS.EmployerIncentives.Functions.DomainMessageHandlers/ServiceCollectionExtensions.cs

```diff
@@ -84,5 +58,50 @@ public static IServiceCollection AddCommandService(this IServiceCollection servi
 
             return serviceCollection;
         }
+
+        public static IServiceCollection AddNServiceBus(
+            this IServiceCollection serviceCollection,
+            IConfiguration configuration)
+        {
+            var webBuilder = serviceCollection.AddWebJobs(x => { });
+            webBuilder.AddExecutionContextBinding();
+
+            var endpointConfiguration = new EndpointConfiguration("SFA.DAS.EmployerIncentives.Functions.DomainMessageHandlers")
```

Comment:
hardcoded assembly name

Reply:
Nope just an endpoint name.  Could be anything.